Tests/Show/ShowMapperTest.php

```diff
@@ -66,9 +71,11 @@ public function setUp()
             ->will($this->returnValue(array()));
 
         $this->groups = array();
+        $this->listShowFields = array();
 
         // php 5.3 BC
```

Comment:
@core23 do you think we should remove this in a seperate PR? this looks weird to me